content/tools/plugins/JenkinsPlugin.md

```diff
@@ -4,70 +4,89 @@ description: "Sample Jenkins plugin using the Contrast Java SDK"
-* Enter the **Contrast URL** (the API URL to your Contrast instance). Use *https://app.contrastsecurity.com/Contrast/api* if you're a SaaS customer; all others use the URL of your Contrast UI (e.g., *https://contrastserver/Contrast/api*). 
-* Enter the [Organization ID](admin-orgsettings.html#apikey). Go to the **user menu > Organization Settings > API tab** to find the Organization UUID of the configured user in the Contrast UI. (You can also copy it from the URL after logging in to the Contrast UI.) 
-* Choose the **Result of a vulnerable build** from the dropdown menu. 
-* Check the box to fail a build if the **application is not found** in the Contrast application. 
-* Choose to allow global Contrast Vulnerability Security Controls to be overridden in post-build actions. (See the **Contrast Vulnerability Security Controls**
-section for more details.)
-* Click the **Apply** button to save the profile settings.
+3.  Enter your [Contrast Username](user-account.html#loginpwd). Your username is usually the email address you use for your account in Contrast. 
+
+4. Enter your [Contrast API Key](user-account.html#profile) and [Contrast Service Key](user-account.html#profile), obtainable from the **Your Account > Profile > Your Keys** section of your Contrast instance.        
+
+5. Enter your **Contrast URL** (the API URL to your Contrast instance). Use *https://app.contrastsecurity.com/Contrast/api* if you're a SaaS customer; all others use the URL of your Contrast UI (e.g., *https://contrastserver/Contrast/api*).
+
+6. Enter your [Organization ID](admin-orgsettings.html#apikey), also obtainable from the **Your Account > Profile > Your Keys** section of your Contrast instance.
+
+7. Choose to have Contrast mark your Jenkins job when your application is deemed too vulnerable with one of the following options from the dropdown menu: **Result of a vulnerable build**
+
+8. If your Jenkins instance is not able to find your application, you may choose to have your Jenkins job automatically fail by checking the box: **Fail build if application is not found**
+
+9. In a moment, you will define at the Jenkins _system_ level the criteria that Contrast will use to determine whether your application is too vulnerable. You may elect to have individual jobs overriding your criteria with their own by checking the box **Allow job level application vulnerability security controls to override those controls set here at the system level**. To enforce consistency of criteria across all Jenkins job in your instance, do not check this box.
+
+10. Click the **Apply** button to save your connection information.
+
+11. Click the **Test Contrast Connection** button to make sure that the plugin will be able to authenticate to Contrast and retrieve information about your applications' vulnerabilities.
+
+
+### Defining Vulnerability Security Controls (Global Scope)
```

Comment:
@contrast-product I'm going to change Global Scope to System Level as we reference the scope as "System Level" in the plugin